src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,125 @@
+@import "utils/variables";
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: 944px;
+  box-sizing: content-box;
+  &__card {
+    margin: 0 48px 46px 0;
+    padding: 32px 16px 16px;
+  }
+}
+
+.card {
+  max-width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+  &__link {
+    display: block;
+  }
+  &__img {
+    display: block;
+    max-width: 100%;
+    height: 134px;
+    padding: 0 3px;
+    margin-bottom: 40px;
+    text-align: center;
+    transition: transform 0.5s;
```

Create variable for transition duration
```suggestion
    transition: transform $transitionDuration;
```